spec/javascript/controllers/talk-to-us_controller_spec.js

```diff
@@ -45,11 +45,6 @@ describe('TalkToUsController', () => {
             expect(chat.style.display).toContain('inline-block');
         });
 
-        it("should make the tta section have 48% width", () => {
```

Comment:
How come this test is being removed?

Reply:
Didnt realise on a previous PR when I changed the talks section styling that we no longer set the width to 48%. using flex it fills the space needed.